public/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="./logo.png" type="image/x-icon">
+  <title>pick your movie</title>
+</head>
+<body>
+  <header>  
+    <img src="./logo.png" alt="logo">
+    <div class="search-box">
+      <input type="input" class="search" id="search" placeholder="type your movie name here...">
+      <i class="fa-solid fa-x empty-icon"></i>
+      <i class="fa-solid fa-magnifying-glass search-icon"></i>
+    </div>
+    <section class="autocomplete"></section>
+  </header>
+  <section class="cards-container"></section>
+  <script src="./script.js"></script>
+</body>
+</html>
```

Comment:
it is a good style to always put a newline at the end of the file

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log